galasa-ui/src/components/TokenRequestModal.tsx

```diff
@@ -49,9 +51,12 @@ export default function TokenRequestModal() {
       console.error('Failed to request a personal access token: %s', err);
     }
   };
+    
   return (
     <>
-      <Button onClick={() => setOpen(true)}>Request personal access token</Button>
+      <Button iconDescription={"Create new access token"} data-testid="token-request-btn" disabled={isDisabled} hasIconOnly onClick={() => setOpen(true)}>
+          <Add/>
```

Comment:
The purpose of the "+" isn't immediately obvious compared to explicitly saying something like "Request personal access token", but happy to keep as is if this has already been discussed.